_dashboards/discover/index-discover.md

```diff
@@ -1,104 +1,115 @@
 
-![Analyzing data in the document table]({{site.url}}{{site.baseurl}}/images/discover-analyze.png)
+To filter your data:
 
-## Saving the search
+1. Under the DQL search bar, choose **Add filter**.
+1. Select the desired options from the **Field**, **Operator**, and **Value** dropdown lists. For example, select `Cancelled`, `is`, and `true`.
+1. Choose **Save**.
+1. To remove the filter, choose the cross icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/cross-icon.png" class="inline-icon" alt="cross icon"/>{:/}) next to the filter name.
+1. Add more filters to further explore the data.  
+
+## Save a search
 
-Saving a search saves the query text, filters, and current data view. To save your search to use it later, generate a report, or build visualizations and dashboards:  
+To save your search, including the query text, filters, and current data view, follow these steps:  
 
-1. Choose the save icon in the toolbar. 
+1. Select **Save** in the upper-right area. 
 1. Give the search a title, and then choose **Save**. 
-1. Choose the save icon to access the saved search, as shown in the following image. 
+1. Select **Open** to access the saved search. 
+
+## Create visual representations of your data discoveries
 
-<img src="{{site.url}}{{site.baseurl}}/images/discover-save.png" alt= "Save search interface" width="350" height="350">
+To create visualizations of your data findings, follow these steps:
 
-## Visualizing the search
+1. Select the inspect icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/inspect-icon.png" class="inline-icon" alt="inspect icon"/>{:/}) next to the field you want to visualize. An example is shown in the following image.
 
-You can quickly visualize an aggregated field from **Discover**:
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/discover-visual.png" alt="Visualize button in Discover" width="400"/>
 
-1. From the **Available fields** list, select `FlightDelayType` and then choose **Visualize**, as shown in the following image.
+2. Select the **Visualize** button. A visualization is displayed. An example is shown in the following image.
 
-<img src="{{site.url}}{{site.baseurl}}/images/discover-visualize.png" alt= "Visualizing search queries from Discover" width="350" height="350">
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/visualize-discover.png" alt="Visual representation of data" width="700"/>
 
-Dashboards creates a visualization for this field, which in this case is a basic bar chart, as shown in the following image.
+## Set alerts
 
-<img src="{{site.url}}{{site.baseurl}}/images/discover-visualize-2.png" alt= "Bar chart created from Discover" width="600" height="600">
+You can set alerts to notify you when your data changes beyond the thresholds you define. Learn more about using **Discover** to set up [alerting dashboards and visualizations]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/dashboards-alerting/).
```

Comment:
The second sentence is "floating." Should it just be "To learn more about using Discover to set up alerting dashboards and visualizations, see [link]"?

Reply:
Revised.